sensor_msgs/Image image
---
sensor_msgs/Image processed_image
